frontend/packages/kubevirt-plugin/integration-tests/tests/models/wizard.ts

```diff
@@ -129,7 +129,11 @@ export class Wizard {
   async addNIC(nic: Network) {
     await click(view.addNICButton);
     const addNICDialog = new NetworkInterfaceDialog();
-    await addNICDialog.create(nic);
+    const err = await addNICDialog.create(nic);
```

Comment:
can we do just return here?

Reply:
I can't do something like `return await addNICDialog.create(nic);` here, and without `await` it will not work well

Reply:
why will it not work well without await? have you tried it?

Reply:
please simplify all the returns. I don't see a reason why this wouldn't work. I tried to simulate it and it works for me: [example]( https://www.typescriptlang.org/play?#code/FAYw9gdgzgLgBAWwJ4EkIQKYCcCCUkQgBiArhHALxwCG+hcAFAJSUB8cA3sHD3FhjBJZyAJgAMwAL7BQkWIiR4CxMpRp0QjFhXZdeccNHj8oJADbwq1AO7UAlvGRpMuDaQjMA3DP12AZowm5jAsevq8-ILCfBimFt760twRAkLkEOZm3kmG8shKhO4AInb8IJbqylpsnMk8kWkKztgFKh5M2TLA+W5kzAB0MAAWGB65YGYY-WZgAOZM3Yq9ECVlMAPDowzjk9NzTEA)

Reply:
Done